.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+# Copyright (C) Nitrokey GmbH
+# SPDX-License-Identifier: CC0-1.0
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Install rust
+      run: |
+        rustup show
```

```suggestion
        rustup show active-toolchain || rustup toolchain install
```

see https://github.com/rust-lang/rustup/blob/master/CHANGELOG.md#1280---2025-03-04